server/services/prisonerProfileService.test.ts

```diff
@@ -169,144 +150,127 @@ describe('Prisoner profile service', () => {
             { text: 'Booked', attributes: { 'data-test': 'tab-upcoming-status' } },
           ],
         ],
-        pastVisits: [
-          [
-            {
-              html: "<a href='/visit/ab-cd-ef-gh'>ab-cd-ef-gh</a>",
-              attributes: { 'data-test': 'tab-past-reference' },
-            },
-            { html: 'Social', attributes: { 'data-test': 'tab-past-type' } },
-            { text: 'Hewell (HMP)', attributes: { 'data-test': 'tab-past-location' } },
-            {
-              html: '<p>17 August 2022<br>10:00am - 11:00am</p>',
-              attributes: { 'data-test': 'tab-past-date-and-time' },
-            },
-            { html: '<p>Mary Smith</p>', attributes: { 'data-test': 'tab-past-visitors' } },
-            { text: 'Booked', attributes: { 'data-test': 'tab-past-status' } },
-          ],
-        ],
+        prisonerDetails: {
+          offenderNo: 'A1234BC',
+          name: 'Smith, John',
+          dob: '2 April 1975',
+          convictedStatus: 'Convicted',
+          category: 'Cat C',
+          location: '1-1-C-028',
+          prisonName: 'Hewell (HMP)',
+          incentiveLevel: 'Standard',
+          visitBalances: {
+            remainingVo: 1,
+            remainingPvo: 2,
+            latestIepAdjustDate: '21 April 2021',
+            latestPrivIepAdjustDate: '1 December 2021',
+          },
+        },
       })
     })
-
-    it('Does not look up visit balances for those on REMAND', async () => {
-      const inmateDetail = TestData.inmateDetail({ legalStatus: 'REMAND' })
-      const prisoner = TestData.prisoner()
-
-      const bookings = <PagePrisonerBookingSummary>{
-        content: [
-          {
-            bookingId: 22345,
-            bookingNo: 'B123445',
-            offenderNo: inmateDetail.offenderNo,
-            firstName: inmateDetail.firstName,
-            lastName: inmateDetail.lastName,
-            dateOfBirth: inmateDetail.dateOfBirth,
-            agencyId: 'HEI',
-            legalStatus: 'REMAND',
-            convictedStatus: 'Remand',
-          },
-        ],
-        numberOfElements: 1,
+    // Skipped - previously used endpoints were skipped if prisoner was on remand, this logic may wish be to included in the new endpoint
+    it.skip('Does not return visit balances for those on REMAND', async () => {
```

Comment:
👍 OK, makes sense to `skip` this for now and we can do some testing to ensure this scenario is handled properly by front-end or orchestration service